packages/composites/the-graph/test/integration/adapter.test.ts

```diff
@@ -62,6 +62,17 @@ describe('execute', () => {
           } 
         },
       },
```

Comment:
It seems the old tests here are not passing because the "dex" param was not added for them. Could you please also include that in this PR, so we can get all tests passing? :slightly_smiling_face: